tests/ExamplesTest.php

```diff
@@ -11,6 +11,10 @@ public function setUp(): void
     {
         parent::setUp();
 
+        if ($this->isShardedCluster()) {
+            $this->markTestSkipped('Examples are not tested on sharded clusters.');
+        }
```

Comment:
Obligatory @amschwerin:

![20160923_andy_sharding](https://user-images.githubusercontent.com/244663/203211525-c5066b71-a2b7-46ef-844d-c2a05eadd7e6.jpg)

I am a creature of habit: https://github.com/mongodb/specifications/pull/565#issuecomment-501395705

Reply:
I tried hard to keep those tests in, but unfortunately some tests failed on sharing due to a server error ("no progress was made executing batch op after 5 rounds"). Since we're only dealing with examples here I opted against spending more time on testing against sharded clusters and gave up trying.